zebra-rpc/src/methods/get_block_template.rs

```diff
@@ -139,8 +150,106 @@ where
         }
         .boxed()
     }
+
+    fn get_block_template(&self) -> BoxFuture<Result<GetBlockTemplate>> {
+        async move {
+            let empty_string = String::from("");
+
+            // Returns empty `GetBlockTemplate`
+            Ok(GetBlockTemplate {
+                capabilities: vec![],
+                version: 0,
+                previousblockhash: empty_string.clone(),
+                blockcommitmentshash: empty_string.clone(),
+                lightclientroothash: empty_string.clone(),
+                finalsaplingroothash: empty_string.clone(),
+                defaultroots: DefaultRoots {
+                    merkleroot: empty_string.clone(),
+                    chainhistoryroot: empty_string.clone(),
+                    authdataroot: empty_string.clone(),
+                    blockcommitmentshash: empty_string.clone(),
+                },
+                transactions: vec![],
+                coinbasetxn: Coinbase {},
+                longpollid: empty_string.clone(),
```

Comment:
Zebra doesn't support the "long polling" feature, so we shouldn't include the `longpollid` field.
(It's an optional `getblocktemplate` feature.)

Reply:
Here's the spec:
https://en.bitcoin.it/wiki/BIP_0022#Optional:_Long_Polling

Reply:
That is one less to worry about, thanks! I removed it at https://github.com/ZcashFoundation/zebra/pull/5462/commits/5756a77f89d8597ce146da071878d978c4706711